tools/config.schema.yaml

```diff
@@ -0,0 +1,855 @@
+                      properties:
+                        missed:
+                          type: array
+                          uniqueItems: True
+                          items:
+                            type:
+                              string
+                            enum: [count, distinctcontacts, timefirstcall, timelastcall, countmostfrequentcontact]
+                        incoming:
+                          type: array
+                          uniqueItems: True
+                          items:
+                            type:
+                              string
+                            enum: [count, distinctcontacts, meanduration, sumduration, minduration, maxduration, stdduration, modeduration, entropyduration, timefirstcall, timelastcall, countmostfrequentcontact]
+                        outgoing:
+                          type: array
+                          uniqueItems: True
+                          items:
+                            type:
+                              string
+                            enum: [count, distinctcontacts, meanduration, sumduration, minduration, maxduration, stdduration, modeduration, entropyduration, timefirstcall, timelastcall, countmostfrequentcontact]
+        additionalProperties:
+          $ref: "#/definitions/PROVIDER"
+
+  PHONE_CONVERSATION:
+    type: object
+    required: [TABLE, PROVIDERS]
+    properties:
+      TABLE:
+        type: object
+        required: [ANDROID, IOS]
+        properties:
+          ANDROID:
+            type: string
+          IOS:
+            type: string
+      PROVIDERS:
+        type: ["null", object]
+        properties:
+          RAPIDS:
+            allOf:
+              - $ref: "#/definitions/PROVIDER"
+              - properties:
+                  FEATURES:
+                    items:
+                      type: string
+                      enum: ["minutessilence", "minutesnoise", "minutesvoice", "minutesunknown","sumconversationduration","avgconversationduration",
+    "sdconversationduration","minconversationduration","maxconversationduration","timefirstconversation","timelastconversation","noisesumenergy",
+    "noiseavgenergy","noisesdenergy","noiseminenergy","noisemaxenergy","voicesumenergy",
+    "voiceavgenergy","voicesdenergy","voiceminenergy","voicemaxenergy","silencesensedfraction","noisesensedfraction",
+    "voicesensedfraction","unknownsensedfraction","silenceexpectedfraction","noiseexpectedfraction","voiceexpectedfraction",
+    "unknownexpectedfraction","countconversation"]
+                  RECORDING_MINUTES: 
+                    type: integer
+                  PAUSED_MINUTES:
+                    type: integer
+        additionalProperties: 
+          $ref: "#/definitions/PROVIDER"
+  
+  PHONE_DATA_YIELD:
+    type: object
+    required: [SENSORS, PROVIDERS]
+    properties:
+      SENSORS:
+        type: ["null", array]
+        items:
+          type: string
```

Comment:
SENSORS should be an enum with possible values equal to any PHONE_* sensor except for PHONE_DATA_YIELD itself

Reply:
fixed

Reply:
the new sensors are missing like PHONE_APPLICATIONS_CRASHES